src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+public class Parser {
+
+    /**
+     * Parses text the user inputs into first word and the rest.
+     *
+     * @param input user input.
+     * @return Array with 2 elements containing first word and the rest.
+     */
+    public static String[] parseUserInput(String input) {
+        String[] cmdArgs = input.split(" ", 2);
+        return stripStrings(cmdArgs);
+    }
+
+    /**
+     * Parses task argument into task name and time.
+     *
+     * @param args Task argument.
+     * @param delim Delimiter used to split the input.
+     * @return Array with 2 elements containing task name and time.
+     */
+    public static String[] parseArgs(String args, String delim) {
+        String[] nameTime = args.split(delim);
+        checkArg(nameTime);
+        return stripStrings(nameTime);
+    }
```

Comment:
Maybe we can just name the parameters as taskArgument and delimiter. Although it is a bit longer this way, it is more meaningful.